tests/sample-v2-addon/tests/integration/components/navigation-menu-test.gts

```diff
@@ -34,38 +31,26 @@ module('Integration | Component | navigation-menu', function (hooks) {
 
     const links = findAll('[data-test-link]');
 
-    assert.strictEqual(links.length, 4, 'We see 4 links.');
+    assert.strictEqual(links.length, 1, 'We see 1 link.');
 
     assert
       .dom(links[0])
       .hasAttribute('href', '/', 'We see the correct href for the 1st link.')
       .hasText('Home', 'We see the correct label for the 1st link.');
+  });
 
-    assert
-      .dom(links[1])
-      .hasAttribute(
-        'href',
-        '/dashboard',
-        'We see the correct href for the 2nd link.',
-      )
-      .hasText('Dashboard', 'We see the correct label for the 2nd link.');
-
-    assert
-      .dom(links[2])
-      .hasAttribute(
-        'href',
-        '/form',
-        'We see the correct href for the 3rd link.',
-      )
-      .hasText('Form', 'We see the correct label for the 3rd link.');
+  test('CSS modules', async function (assert) {
+    await render(
+      <template>
+        <NavigationMenuNew
+          @menuItems={{array
+            (hash label="Home" route="index")
+          }}
+          @name="Main Navigation"
+        />
+      </template>
+    );
 
-    assert
-      .dom(links[3])
-      .hasAttribute(
-        'href',
-        '/products',
-        'We see the correct href for the 4th link.'
-      )
-      .hasText('Products', 'We see the correct label for the 4th link.');
+    assert.dom('[data-test-link="Home"]').hasClass(/^sample-v2-addon/, 'We see the local style.');
```

Comment:
A new test for `<NavigationMenu>` component. The test asserts that CSS modules can be implemented in v2 addons.